coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +58,21 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        original_line = str(original_file_dict[filename][
```

Comment:
Why are you calling `str` on a `str`-object? :O

Reply:
Sorry. :face_with_head_bandage: Will correct it.